### PR TITLE
Move smoke tests

### PR DESCRIPTION
**Pull request by danieljurek**

### CODE REVIEW

**Comment by danieljurek** [comment]
cc @kaerm

**Review by pakrym — APPROVED** [review]

File: eng/documentation/AuthConflictsBetweenSDKs.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Bridging gap between old and new SDKs
```
Comment (pakrym) [inline eng/documentation/AuthConflictsBetweenSDKs.md]: Why doesn't this show as move?

**Review by pakrym — CHANGES_REQUESTED** [review]
You didn't remove the Doc directory

**Review by KrzysztofCwalina — APPROVED** [review]
Besides the issue identified by @pakrym, the PR looks good.

**Comment by weshaggard** [comment]
@danieljurek please update based on my proposal at https://github.com/Azure/azure-sdk/pull/733.

**Review by pakrym — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

File: eng/pipelines/smoke-tests.yml [hunk 2]
```diff
@@ -39,7 +39,7 @@ jobs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
 
-      - script: dotnet run -p ./samples/SmokeTest/SmokeTest/SmokeTest.csproj --framework $(TestTargetFramework)
+      - script: dotnet run -p ./common/SmokeTest/SmokeTest/SmokeTest.csproj --framework $(TestTargetFramework)
```
Comment (weshaggard) [inline eng/pipelines/smoke-tests.yml]: Missing an "s" in `/common/SmokeTests`

File: common/SmokeTests/SmokeTest/SmokeTest.csproj [hunk 3]
```diff
@@ -6,11 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.4" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.3" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.4" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0-preview.3" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.6.0" />
+    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.5" />
```
Comment (weshaggard) [inline common/SmokeTests/SmokeTest/SmokeTest.csproj]: Any reason to bump these to the next prerelease as opposed to GA?

Comment (danieljurek) [reply common/SmokeTests/SmokeTest/SmokeTest.csproj]: These should really be the old values... I want to keep the bumping separated and in a different PR

**Pull request merged**